scriptworker/task.py

```diff
@@ -31,7 +31,17 @@
 
 log = logging.getLogger(__name__)
 
-KNOWN_TASKS_FOR = ('hg-push', 'cron', 'action', 'github-release')
+KNOWN_TASKS_FOR = (
```

Comment:
Hm. I wonder if we should change this to have a set of known `tasks_for` per `cot-product`. We're likely good for now, since gecko will barf if we try to use github cot verification there.

Reply:
Good idea! Done.